<?xml version="1.0" encoding="UTF-8"?> 
<launch>
  <node pkg="rovio" type="rovio_optimizer_node" name="rovio_optimizer" output="screen">
  <param name="filter_config" value="$(find rovio)/cfg/rovio.info"/>
  <param name="optimized_filter_config" value="$(find rovio)/cfg/rovio_optimized.info"/>
  <param name="camera0_config" value="$(find rovio)/cfg/visensor_cam0.yaml"/>
  <param name="camera1_config" value="$(find rovio)/cfg/visensor_cam1.yaml"/>
  <param name="bagfile" value="/media/anurag/DATA-EXT/optitrack_data/InfiniTAM/drz-rig_2016-04-17-21-29-54_clipped.bag"/>
  <param name="groundtruth_topic" value="/drz_rig/estimated_transform"/>
  <param name="cam0_topic" value="/cam0/image_raw"/>
  <param name="cam1_topic" value="/cam1/image_raw"/>
  <param name="imu_topic" value="/imu0"/>
  </node>
</launch>
